static/js/publisher/form.test.js

```diff
@@ -142,7 +148,7 @@ describe("initForm", () => {
     market.initForm(config, initialState, undefined);
   }
 
-  describe("simple", () => {
+  describe("", () => {
```

Comment:
Why empty label? Could it be `describe("form")` or something?
```suggestion
  describe("form", () => {
```